ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/TokenAuthentication.cpp

```diff
@@ -50,22 +49,15 @@ void TokenAuthentication::componentComplete()
   // find QML MapView component
   m_mapView = findChild<MapQuickView*>("mapView");
   if (!m_mapView)
-      return;
+    return;
 
   m_mapView->setWrapAroundMode(WrapAroundMode::Disabled);
 
-  // Create a map using the topographic basemap
-  m_map = new Map(BasemapStyle::ArcGISTopographic, this);
-  m_map->setInitialViewpoint(Viewpoint(Envelope(-19221397.32591107, -208306.59714691807, -3024411.4154897667, 11939432.83566906, SpatialReference(3857))));
+  m_portal = new Portal(m_portalURL, this);
```

Comment:
nit-pick. there's no reason to wait for component complete to setup `m_portal`, `m_portalItem` and `m_map`. These could all be done in the initializer list. That way they'll be created ready to bet set to the map view in this method.